src/lib.rs

```diff
@@ -68,6 +68,12 @@ pub mod plumbing {
     pub use crate::raw::{EnvPtr, FieldPtr, FromJniValue, IntoJniValue, MethodPtr, ObjectPtr};
     pub use crate::refs::NullJRef;
     pub use crate::to_java::{ToJavaImpl, ToJavaScalar};
+    pub use duchess_macro_rules::{
```

Comment:
I wonder if we should make the whole plumbing module doc(hidden)

Reply:
Maybe. I've been thinking we should also rename it to `semver_unstable` or something more explicit.